### ISSUE TITLE
Smart Random feature

### ISSUE THREAD

**Issue by firolightfog** [schismtracker/schismtracker#221]
Hello,

I wanted to write about this one for so long. But I don't think I did. 
Is it hard to implement a Smart Random feature? See below what I mean.

Usage 1: 
- F3 (Sample list)
- Alt-Z (AdLib sample)
- Alt-R (random values)

But I'd alternatively apply it to other cases too. 
Usage 2:
- F3 (Sample list)
- Alt-Y (Text to sample)
- Alt-R (random values)

It could be applied to any sliders even like below.
Usage 3:
- F3 (Sample list)
- Tab (Jump to Default Volume)
- Alt-R (random value between 0-64)

<strike>I find it less useful but I'd even use for this.
Usage 4:
- F2 (Pattern Editor)
- Alt-D (Quick mark)
- Alt-K (Slide volume values like 21,22,23,24,25)
- Alt-R (Randomize volume like 21,11,33,44,25)
</strike>

What do you think? It would be fun, wouldn't it? :)

Best regards, Andras

**Comment by jangler** [comment]
Ctrl-U, Ctrl-Y, and Ctrl-K do vary volume, panning, and effect parameter in the pattern editor.

**Comment by firolightfog** [comment]
@jangler You are right. "Usage 4" example is obsolete. I start using these combinations right away. :)
Thanks, A.
